### DISCUSSION TITLE
TMAP8 Input files missing

### DISCUSSION THREAD

**Discussion by libinbabu1991** [idaholab/TMAP8#198] [Q&A] [answered]
Hello @simopier , 
I read your paper "Tritium Transport Modeling at the Pore Scale in Ceramic Breeder Materials Using TMAP8". I would like to try the code that you have made available in the section VII. Data availability.
I downloaded the entire folder from github and searched for the input files used for the TMAP8 study, but I couldn't locate them.

Could you please help me locate them?

Thanks.

**Comment by simopier** [comment]
Hi @libinbabu1991, 

Thank you for reaching out! I’m glad you’re interested in these simulations.

You are correct that these files are not yet on the GitHub repository.
I’ll clean up these input files, make sure they run well, and add them to the TMAP8 website as an example. Stay tuned!

**Reply by libinbabu1991** [reply]
Thanks! Looking forward to it. 

I wanted some more examples of TMAP8 application. I found some ver-xxx files in the below directory. Are there documentation on what they do?
TMAP8-devel\test\tests

**Reply by simopier** [reply]
Of course! All of our capabilities are documented. You can find the updated list of TMAP8 verification and validation cases here: https://mooseframework.inl.gov/TMAP8/verification_and_validation/index.html
and there is also an example page: https://mooseframework.inl.gov/TMAP8/examples/tmap_index.html (which is where I'll put the description of the simulations published in "Tritium Transport Modeling at the Pore Scale in Ceramic Breeder Materials Using TMAP8").

**Reply by libinbabu1991** [reply]
Thanks, @simopier.

**Reply by libinbabu1991** [reply]
Hi @simopier,
I would like to follow up on the input files.

I also wanted to ask what was the generation term q used in the 2D study mentioned in the second half of the paper. Was it taken from the same experimental reference?

Another question: Can you connect TMAP8 with GUI like NEAMS Workbench?

Thanks,
Libin

**Reply by simopier** [reply]
@libinbabu1991, I wanted to let you know that I recently created a PR to add an example with the 2D input file from this example: 
https://github.com/idaholab/TMAP8/pull/230. It has not been reviewed and merge yet, so it is not yet visible on the website, but you can still find the current input files and the documentation there. 

The 2D study was a simulation of an absorption, so the generation term was set to 0. 

Concerning the NEAMS workbench, great question! We have not explicitly applied it to TMAP8. But since TMAP8 is a MOOSE-based application, it should be applicable, as noted in the table at the bottom of this page: https://www.ornl.gov/onramp/neams-workbench.

**Reply by libinbabu1991** [reply]
Hi @simopier,
Thanks a lot for the files and your answers!

**Reply by simopier** [reply, marked as answer]
Hi @libinbabu1991, 
I wanted to let you know that the mesoscale simulations with TMAP8 was reviewed and approved and is now available as a tested example on the TMAP8 website: https://mooseframework.inl.gov/TMAP8/examples/tmap_index.html

**Reply by libinbabu1991** [reply]
Hello @simopier , thank you for the information. I will review.
